src/main/java/duke/Task.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+/**
+ * The parent of deadline, event, todo
+ */
+abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
I think you don't need the 'this' keyword for isDone unless it's being shadowed by a constructor parameter (eg. if constructor had a param boolean isDone then you would do this.isDone = isDone).